src/components/dialogs/network-modifications/two-windings-transformer/tap-changer-pane/ratio-tap-changer-pane/ratio-tap-changer-pane.js

```diff
@@ -46,6 +46,9 @@ const RatioTapChangerPane = ({
     isModification = false,
 }) => {
     const { trigger } = useFormContext();
+    const watchLRegulationMode = useWatch({
```

Comment:
It's already present below, remove it
And you can remove the associated comment

Reply:
Done